### PR TITLE
Make stream text mouse-selectable

### PR DESCRIPTION
**Pull request by gitnik**

### CODE REVIEW

**Comment by jasonwilson** [comment]
Thanks for the PR @gitnik. This doesn't seem to fix #26 for me-- what terminal and settings are you using?

**Comment by wmertens** [comment]
When using Terminal.app on OS X Mavericks, this does work.

**Comment by wmertens** [comment]
BTW, the graphs get included in the copy/paste. Perhaps they should be in a horizontal section of the screen?
e.g.:
```
Time: 9208ms                                                                                                                               ││     ⡇                                       │
│                               Asset     Size  Chunks             Chunk Names                                                              ││     ⡇                                       │
│                           broker.js  19.7 kB       4  [emitted]  broker                                                                   ││  50 ⡇                                       │
│24bcfa5936332a7f0742f5c61c2f69b6.jpg  7.71 kB          [emitted]                                                                           ││     ⡇                                       │
│                           server.js  8.61 MB       1  [emitted]  server                                                                   ││     ⡇                                       │
│                           worker.js   8.6 MB       2  [emitted]  worker                                                                   ││     ⡇                                       │
│                             init.js    20 kB       3  [emitted]  init                                                                     ││     ⡇                                       │
│           0_18d892acb8fe558a88ab.js  40.7 kB       0  [emitted]                                                                           ││  25 ⡇                                       │
│       0_18d892acb8fe558a88ab.js.map  43.9 kB       0  [emitted]                                                                           ││     ⡇                                       │
│                       server.js.map   8.4 MB       1  [emitted]  server                                                                   ││     ⡇                                       │
│                       worker.js.map   8.4 MB       2  [emitted]  worker                                                                   ││     ⡇                                       │
│                         init.js.map  21.3 kB       3  [emitted]  init                                                                     ││     ⡇                                       │
│                       broker.js.map  21.1 kB       4  [emitted]  broker                                                                   ││  0  ⡇⠤⠤⠤⠤⠤⠤⠤⠤⠤⠤⠤⠤⠤⠤⠤⠤⠒⠒⠒⠒⠤⠤⠤⠤⠤⠤⠤⠤⠤⠤⠤⠤⠄      │
│ [415] multi server 40 bytes  [built]                                                                                                      ││     ⠓⠒⠒⠒⠒⠒⠒⠒⠒⠒⠒⠒⠒⠒⠒⠒⠒⠒⠒⠒⠒⠒⠒⠒⠒⠒⠒⠒⠒⠒⠒⠒⠒⠒⠒⠒    │
│ [416] multi worker 64 bytes  [built]                                                                                                      ││      9  8   7  6   5   4  3   2  1   0      │
│    + 420 hidden modules                                                                                                                   ││                                             │
```

**Comment by wmertens** [comment]
a really annoying problem is that with this PR the scrolling of the boxes no longer works.

File: lib/views/stream-view.js [hunk 1]
```diff
@@ -17,7 +17,7 @@ class StreamView {
         inverse: true
       },
       keys: true,
-      mouse: true,
+      mouse: false,
```
Comment (daithi-coombes) [inline lib/views/stream-view.js]: brilliant... finally. +1 I can now select stdout/err :+1:

**Comment by wmertens** [comment]
Would it be possible to turn this on/off with a key? Not having scrollback is very limiting…

**Comment by wearhere** [comment]
Here's how to re-enable scrolling while also keeping text selection enabled: https://github.com/chjj/blessed/issues/263#issuecomment-392290076

**Pull request closed without merging**